packages/babel-parser/src/parse-error/credentials.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+
+export const ParseErrorCodes = Object.freeze({
+  SyntaxError: "BABEL_PARSER_SYNTAX_ERROR",
+  SourceTypeModuleError: "BABEL_PARSER_SOURCETYPE_MODULE_REQUIRED",
+});
+
+export type ParseErrorCode = $Values<typeof ParseErrorCodes>;
```

Comment:
I'm surprised that Flow reports this as `$Values<{| SourceTypeModuleError: string, SyntaxError: string |}>`, but then it does the correct thing and doesn't allow me to assign any string to it.